deploy/deploy-image.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# This script captures the steps required to successfully
+# deploy the image plugin driver.  This should be considered
+# authoritative and all updates for this process should be
+# done here and referenced elsewhere.
+
+# The script assumes that kubectl is available on the OS path 
+# where it is executed.
+
+set -e
+set -o pipefail
+
+BASE_DIR=$(dirname "$0")
+K8S_RELEASE=${K8S_RELEASE:-"release-1.13"}
```

Comment:
Does this actually work on 1.13?

Reply:
ah. no. it doesn't actually work in 1.14.[0-1] either...

Reply:
I guess it's not used anymore, so we can remove this variable?